include/FileBrowser.h

```diff
@@ -87,6 +87,8 @@ class FileBrowserTreeWidget : public QTreeWidget
 	FileBrowserTreeWidget( QWidget * parent );
 	virtual ~FileBrowserTreeWidget() = default;
 
+	QList<QString> expandedDirs( QTreeWidgetItem * item=NULL ) const;
```

Comment:
Coding conventions: "Infix operators ('=', '+', '-', '*', '/', etc.) should have a space before and after them"
```suggestion
	QList<QString> expandedDirs( QTreeWidgetItem * item = nullptr ) const;
```